sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+---
+friendly_name: Data from external source Adjust
+description: >
+  This table represents data downloaded from Adjust. Please note that the API token used is Marlene Hirose's personal Adjust API token
+owners:
+  - mhirose@mozilla.com
+labels:
+  incremental: false
+scheduling:
+  dag_name: bqetl_adjust_derived
+  date_partition_parameter: submission_date
+  parameters:
+bigquery:
+  time_partitioning:
+    type: day
+    field: submission_date
+    require_partition_filter: false
+external_data:
+  format: csv
+  source_uris:
```

Comment:
Should I add the Adjust API source URIs here? There are 11 with the APP tokens - we don't want to add those tokens to git

Reply:
`external_data` can be removed here. It's only used in cases where you simply import a CSV file into a BigQuery table. In your case you have a script with more complex logic that handles the import